hdf5.spec

```diff
@@ -205,7 +204,6 @@ HDF5 tests
 
 %prep
 %setup -q -a 2 -n %{name}-%{version}%{?snaprel}
-%patch10 -p1 -b .hdf5-1.10.5-11.4-g07066a381e
 %patch0 -p1 -b .LD_LIBRARY_PATH
```

Comment:
Looks like this patch is failing to apply to the 1.10.5 source.  Might need to be regenerated or massaged.